packages/playground/wordpress/src/mu-plugins/auto_login.php

```diff
@@ -106,7 +106,7 @@ function playground_auto_login() {
 	/**
 	 * Check if the request is for the login page.
 	 */
-	if (false !== stripos( wp_login_url(), $_SERVER['SCRIPT_NAME'] )) {
+	if (false === stripos( wp_login_url(), $_SERVER['SCRIPT_NAME'] )) {
 		return;
```

Comment:
What would happen without this check? Also, stripos doesn't seem like a reliable way of testing for a login page. What would a rigorous, 100% reliable check look like?

Reply:
This code is a copy of [is_login](https://developer.wordpress.org/reference/functions/is_login/) U just extracted it because I need to add `empty($_SERVER['SCRIPT_NAME'])`.

> What would happen without this check?

`playground_auto_login` would be called more frequently and users would be redirected on every page load.
But this makes me think that it could all be one hook. I will explore this a bit more and at least add more comments.